### PR TITLE
Add Ghost Browser.app v1.0.3.6

### PR DESCRIPTION
**Pull request by dvdoliveira**
After making all changes to the cask:

- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/ghost-browser.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'ghost-browser' do
+  version '1.0.3.6'
+  sha256 'c04f6f1e3c64c20d4d9fad06e1620454f64f04a5c1603664190c2189c947ed0e'
+
+  url 'https://ghostbrowser.com/ghostdev/downloads/GhostBrowser-1.0.3.6.dmg'
```
Comment (miccal) [inline Casks/ghost-browser.rb]: Please change this to `url "https://ghostbrowser.com/ghostdev/downloads/GhostBrowser-#{version}.dmg"`

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/ghost-browser.rb (hunk 1 above)
Comment (dvdoliveira) [inline Casks/ghost-browser.rb]: Sure! Sorry about that.

**Comment by dvdoliveira** [comment]
Done!

File: Casks/ghost-browser.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'ghost-browser' do
+  version '1.0.3.6'
+  sha256 'c04f6f1e3c64c20d4d9fad06e1620454f64f04a5c1603664190c2189c947ed0e'
+
+  url 'https://ghostbrowser.com/ghostdev/downloads/GhostBrowser-#{version}.dmg'
```
Comment (miccal) [inline Casks/ghost-browser.rb]: You need to use double quotes for the interpolation to work, like this:
```
url "https://ghostbrowser.com/ghostdev/downloads/GhostBrowser-#{version}.dmg"
```

**Review by miccal — CHANGES_REQUESTED** [review]

File: Casks/ghost-browser.rb (hunk 2 above)
Comment (dvdoliveira) [inline Casks/ghost-browser.rb]: 👍

**Comment by miccal** [comment]
Thank you.

**Pull request merged**